creme/linear_model/glm.py

```diff
@@ -129,10 +122,9 @@ class LinearRegression(GLM, base.Regressor):
             will be not be updated.
         l2: Amount of L2 regularization used to push weights towards 0.
         clip_gradient: Clips the absolute value of each gradient value.
-        initializer: Weights initialization scheme.
 
     Attributes:
-        weights (collections.defaultdict): The current weights.
+        weights (utils.VectorDict): The current weights.
```

Comment:
I'm not sure, but maybe that `weights` should be a computed property, whilst the `VectorDict` should be stored in a `_weights` attribute. What do you thing?

Reply:
Yes that seems better.